components/modal/__tests__/Modal.test.js

```diff
@@ -44,4 +44,13 @@ describe('Modal', () => {
     const wrapper = mount(<ModalTester footer={null} />);
     expect(wrapper.render()).toMatchSnapshot();
   });
+
+  it('render with very long word fill in title', () => {
+    const wrapper = mount(
+      <ModalTester
+        title="Modal very long title:1:abcdefghijklmnopqrstuvwxyz:it:is:so:long:maybe:over:mask:view"
+      />
+    );
+    expect(wrapper.render()).toMatchSnapshot();
+  });
```

Comment:
This test case is not useful to test work break situation.

Reply:
I dont know what I should do for testing it, and I have offered a problem demo, ex:
[![Edit n3pk656j5p](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/n3pk656j5p)

Reply:
We don't have tests for CSS right now.